Buttplug.Test/Server/Bluetooth/Devices/VorzeBachTests.cs

```diff
@@ -0,0 +1,56 @@
+﻿// <copyright file="VorzeCycloneSATests.cs" company="Nonpolynomial Labs LLC">
+//     Buttplug C# Source Code File - Visit https://buttplug.io for more info about the project.
+//     Copyright (c) Nonpolynomial Labs LLC. All rights reserved. Licensed under the BSD 3-Clause
+//     license. See LICENSE file in the project root for full license information.
+// </copyright>
+
+// Test file, disable ConfigureAwait checking.
+// ReSharper disable ConsiderUsingConfigureAwait
+
+using System.Diagnostics.CodeAnalysis;
+using System.Threading.Tasks;
+using NUnit.Framework;
+
+namespace Buttplug.Server.Test.Bluetooth.Devices
+{
+    [SuppressMessage("StyleCop.CSharp.DocumentationRules", "SA1600:ElementsMustBeDocumented", Justification = "Test classes can skip documentation requirements")]
+    [TestFixture]
+    public class VorzeBachTests : VorzeSATests
+    {
+        private string _deviceName = "Bach smart";
+        private byte _commandPrefix = 0x06;
```

Comment:
Yeah... I'm not sure I should have made it an enumeration at all.

It fits for these 3 devices, but the piston looks like it might use the commandId as a position value. The stop value for the piston also uses 0x01 for the command value, so I'm really not sure that the enum will make as much sense soon.

Reply:
Eh, for now it's fine. I'm ok kicking the can down the road for Piston support until people actually have Pistons.